code/modules/integrated_electronics/core/assemblies.dm

```diff
@@ -651,6 +654,26 @@
 	max_components = IC_MAX_SIZE_BASE / 2
 	max_complexity = IC_COMPLEXITY_BASE / 2
 
+/obj/item/electronic_assembly/small/implant
+	name = "electronic implant"
+	icon_state = "setup_implant"
+	desc = "It's a case, for building tiny-sized, implantable electronics with."
+	var/obj/item/implant/integrated_circuit/implant = null
+
+/obj/item/electronic_assembly/small/implant/after_printing()
+	. = ..()
+	implant = new(get_turf(src))
+	implant.IC = src
+	implant.icon_state = icon_state
+	usr.dropItemToGround(src)
```

Comment:
🤢 
the unironic use of `usr` outside of a `topic()` call is something that should *never* be done under any circumstance